docs/adr/adr-005-plugins.md

```diff
@@ -89,7 +89,7 @@ func NewBridgeCommand(plugs []node.Plugin) *cobra.Command {
 }
 ```
 
-When starting or initiating a new celestia node, the plugins are passed as settings
+When starting or initiating a new сelestia node, the plugins are passed as settings
```

Comment:
What?

Reply:
sorry for the inconvenience, that was an accidental fix.